applications_tests/lethe-fluid/heat_transfer_laser.prm

```diff
@@ -32,6 +32,7 @@ end
 
 subsection laser parameters
   set enable               = true
+  set type                 = exponential decay
```

Comment:
generally parameters don't have space as far as I remember, but I am not sure? Do we have parameter values with spaces?